olm-catalog/serverless-operator/serverless-operator.package.yaml

```diff
@@ -1,4 +1,8 @@
 packageName: serverless-operator
 channels:
 - name: techpreview
+  currentCSV: serverless-operator.v1.5.0
```

Comment:
This should be 1.6.0 now as well.

Reply:
Does it make sense for our GA release to be in the `techpreview` channel? At some point, I imagine we'll need to change our test scripts to use the `4.3`, `4.4`... or even a new `dev`/`nightly` channel. Question is when that should happen. Thoughts?

Reply:
Maybe. Seems like our scripts need adjustments though wrt. these changes.

Reply:
I'll update 1.5.0 -> 1.6.0 in techpreview channel, since 1.6.0 is still TP.